edge/cmd/edgecore/app/server.go

```diff
@@ -54,18 +54,18 @@ offering HTTP client capabilities to components of cloud to reach HTTP servers r
 			flag.PrintFlags(cmd.Flags())
 
 			if errs := opts.Validate(); len(errs) > 0 {
-				fmt.Fprintf(os.Stderr, "%v\n", utilerrors.NewAggregate(errs))
+				util.PrintByLine(os.Stderr, errs)
```

Comment:
we can use `klog.Fatal` like k8s: https://github.com/kubernetes/kubernetes/blob/fdb2cb4c8832da1499069bda918c014762d8ac05/cmd/kubelet/app/server.go#L164

Reply:
great, so the PrintByLine only splice the errors